.github/workflows/test.yml

```
@@ -95,3 +95,47 @@ jobs:
       - name: Build
         run: pnpm run build
         working-directory: web
+      # - name: Install Playwright
+      #   run: pnpm run e2e-install
+      #   working-directory: web
+      # - name: Run Playwright tests
+      #   run: pnpm run e2e
+      #   working-directory: web
+      #   env:
+      #     # This might depend on your test-runner/language binding
+      #     PLAYWRIGHT_TEST_BASE_URL: djmoxvh6xpra8.cloudfront.net/
+  playwright-tests:
+    # needs: [test-frontend]
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    # if: github.event.deployment_status.state == 'success'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 7.1.5
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+          # cache: 'pnpm'
+      # Should be installed from dependent test-frontend
+      # - name: Install dependencies
+      #   run: pnpm ci
+      #   working-directory: web
+      - name: Install Deps
+        run: pnpm i
+        working-directory: web
+      - name: Install Playwright
+        run: pnpm run e2e-install
+        working-directory: web
+      - name: Run Playwright tests
+        run: pnpm run e2e
+        working-directory: web
+        env:
+          TESTING_DOMAIN: '${{ secrets.TESTING_DOMAIN}}' # @TODO find a way to pass this between deployment
```

I've now merged the changes from #32 which fixes the custom URLs up! You can now use:

```suggestion
          TESTING_DOMAIN: '${{ steps.branch.outputs.name }}.test.granted.run'
```